web/assets/js/views/PhysicalEvaluation-0.3-ehr.js

```diff
@@ -89,10 +89,12 @@ PMI.views['PhysicalEvaluation-0.3-ehr'] = Backbone.View.extend({
         } else {
             firstReading.find('select').val('');
         }
-        // Enable first reading fields except protocol modification field and EHR date field
-        firstReading.find('input, input:checkbox').not('#form_' + field + '-ehr-date').each(function () {
-            $(this).attr('disabled', false);
-        });
+        // If the form is not locked enable first reading fields except protocol modification field and EHR date field
+        if (!parseInt($('form').attr('data-locked'))) {
```

Comment:
Since this is a data attribute, would `.data('locked')` make more sense?

Also, could we use `this.$('form')` to make sure we are only targetting the form in this view? (Or select by id?)